be/src/vec/sink/delta_writer_v2_pool.cpp

```diff
@@ -32,10 +32,15 @@ DeltaWriterV2Map::~DeltaWriterV2Map() = default;
 
 DeltaWriterV2* DeltaWriterV2Map::get_or_create(
         int64_t tablet_id, std::function<std::unique_ptr<DeltaWriterV2>()> creator) {
```

Comment:
Please make sure， not use raw pointer any more.
And add ENABLE_FACTORY_CREATOR to DeltaWriterV2 class.